v0/pipeline/steps/cd/k8s-istio-provider-spec.yaml

```diff
@@ -23,4 +23,10 @@ allOf:
           - type: string
             pattern: "^<\\+input>((\\.)((executionInput\\(\\))|(allowedValues|default|regex)\\(.+?\\)))*$"
             minLength: 1
+      delegateService:
+        oneOf:
+          - type: boolean
+          - type: string
+            pattern: "^<\\+input>((\\.)((executionInput\\(\\))|(allowedValues|default|regex)\\(.+?\\)))*$"
```

Comment:
This pattern supports only runtime input, we also want to support expressions

Reply:
Done